.changeset/fifty-buses-hope.md

```diff
@@ -0,0 +1,23 @@
+---
+"@comet/admin": major
+---
+
+Fix styling of `Alert` according to Comet DXP design
```

Comment:
This is more of an adaptation to the updated design not really a fix imo. 

```suggestion
Adapt the styling of `Alert` to match the updated Comet design
```

Reply:
3de29d58effcc691c280cfea2e79fd98dd4f23e0